src/Tizen.TV.UIControls.Forms/Renderer/MediaPlayerImpl.cs

```diff
@@ -20,41 +20,61 @@
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 using Tizen.Multimedia;
+using Tizen.TV.Multimedia;
 using Tizen.TV.UIControls.Forms.Renderer;
 using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform.Tizen;
-
+using TVM = Tizen.TV.Multimedia;
 [assembly: Xamarin.Forms.Dependency(typeof(MediaPlayerImpl))]
 namespace Tizen.TV.UIControls.Forms.Renderer
 {
     public class MediaPlayerImpl : IPlatformMediaPlayer
     {
-        Player _player;
-
+        TVM.Player _player;
         bool _cancelToStart;
         DisplayAspectMode _aspectMode = DisplayAspectMode.AspectFit;
         Task _taskPrepare;
         TaskCompletionSource<bool> _tcsForStreamInfo;
         IVideoOutput _videoOutput;
         MediaSource _source;
+        DRMManager _drmManager;
 
         public MediaPlayerImpl()
         {
-            _player = new Player();
+            _player = CreateMediaPlayer();
             _player.PlaybackCompleted += OnPlaybackCompleted;
             _player.BufferingProgressChanged += OnBufferingProgressChanged;
         }
 
-        public bool UsesEmbeddingControls
+        protected virtual TVM.Player CreateMediaPlayer()
         {
-            get; set;
+            return new TVM.Player();
         }
 
-        public bool AutoPlay { get; set; }
+        public DRMManager DRMManager
+        {
+            get => _drmManager;
+            set
+            {
+                _drmManager = value;
+                if (value != null)
+                {
+                    _player.SetDrm(value);
+                }
```

Comment:
Why not set null?, is it not allow on PlatformAPI?

Reply:
Calling setDRM(null) raises NullReferencesException, therefore, you should not call this after closing the DRMManager.
The framework maintainer confirmed that a 'player' works well on all contents even when the DRMManager is closed.

Reply:
How about disposed DRMManager?

Reply:
I forgot to mention that, the framework maintainer confirmed that a 'player' works well on all contents even when the DRMManager is closed and 'disposed'.